prep.sh

```diff
@@ -40,7 +40,7 @@ do
 done
 
 # TODO find a way to encode this in some POM so that it can be managed by Dependabot
-version=0.2.1
+version=0.3.0
 timestamp=20190926.172941-2 # TODO https://github.com/jenkinsci/plugin-compat-tester/pull/198
 pct=$HOME/.m2/repository/org/jenkins-ci/tests/plugins-compat-tester-cli/${version}-SNAPSHOT/plugins-compat-tester-cli-${version}-${timestamp}.jar
```

Comment:
Need to revert the `timestamp` stuff added in 453f3d3b4d4444c77cfa62f42f7636fad1883a16 to pick up https://github.com/jenkinsci/plugin-compat-tester/pull/198 which I waited for this release.

Reply:
Just removed it. Of a more robust implementation, there might be a follow-up